Solution.yaml

```diff
@@ -27,26 +27,46 @@ parameters:
       fr: 'Symbole de la monnaie'
       en: 'Currency symbol'
     varType: 'enum'
+    options:
+      tooltipText:
+        fr: "Choisissez un\nsymbole :\n\t$\n\t€\n\t฿\n\t¥"
+        en: "Choose a\nsymbol:\n\t$\n\t€\n\t฿\n\t¥"
```

Comment:
No need for the newline before the "symbol" word. Also maybe we could add dash characters before each enum value ?